template/templates/views/entry.html

```diff
@@ -116,6 +116,10 @@ <h1 dir="auto">
             {{ else }}
                 <time datetime="{{ isodate .entry.Date }}" title="{{ isodate .entry.Date }}">{{ elapsed "UTC" .entry.Date }}</time>
             {{ end }}
+            &centerdot;
```

Comment:
It's an optional feature that users can enable/disable. There is a `if` statement missing. See https://github.com/miniflux/v2/blob/master/template/templates/common/item_meta.html#L10-L16

Reply:
Thanks for the review, I added the missing check.